src/libraries/System.Security.Cryptography.X509Certificates/src/System/Security/Cryptography/X509Certificates/X509Certificate2.cs

```diff
@@ -1068,6 +1076,8 @@ private static bool IsECDiffieHellman(X509Certificate2 certificate)
         /// For PEM-encoded certificates in a file, use <see cref="X509Certificate2(string)" />.
         /// </para>
         /// </remarks>
+        [UnsupportedOSPlatform("ios")]
+        [UnsupportedOSPlatform("tvos")]
```

Comment:
This method should no longer need the attributes, since I don't think it calls anything else that's painted with them.

Reply:
Removed, thanks.